Makefile

```
@@ -1,5 +1,19 @@
 MODELS_DIR="pkg/models"
 
+GO_SOURCE = $(shell find cmd pkg rego -type f -name '*.go') $(shell find pkg rego -type f -name '*.rego')
+CLI_SOURCE = $(GO_SOURCE) go.mod go.sum
+CURRENT_VERSION=$(shell git fetch --tags && git describe --tags --abbrev=0 | sed 's/^v//')
```

```suggestion
CURRENT_VERSION=$(shell sed -n 's/^## v\([0-9.]*\).*/\1/p' CHANGELOG.md | head -n 1)
```

I don't super like the git fetch, it's sort of slow and can prompt me for my ssh keyphrase.  Since this is only used for dev builds, maybe we can use the version from `CHANGELOG.md` which should be the same?

---

This is a good idea! Done in 5bf5549.